### PR TITLE
Add Axona GIN tests

### PR DESCRIPTION
**Pull request by CodyCBakerPhD**
## Motivation

Unless I'm mistaken and it got lost in merge conflicts somehow, it looks like Axona data was never added to the GIN tests despite it being readily available.

This PR adds it and reveals some bugs. Assuming @sbuergers is too busy, who else has experience with the Axona format to help debug this?

## Checklist

- [X] Have you checked our [Contributing](https://github.com/catalystneuro/nwb-conversion-tools/blob/master/docs/contribute.rst) document?
- [X] Have you ensured the PR description clearly describes the problem and solutions?
- [X] Have you checked to ensure that there aren't other open or previously closed [Pull Requests](https://github.com/catalystneuro/nwb-conversion-tools/pulls) for the same change?

### CODE REVIEW

**Comment by alejoe91** [comment]
@CodyCBakerPhD the problem seems to be related to this:
The `get_metadata()` function of the `AxonaRecordingExtractorInterface` loads the experimenter (if available) from the file header and adds it the metadata as `metadata["NWBFile"]["experimenter"] = ['name of experimenter]`.

However, the `deep_dict_update` in the [`get_metadata` of the `NWBconverter` class](https://github.com/catalystneuro/nwb-conversion-tools/blob/add_axona_gin/nwb_conversion_tools/nwbconverter.py#L96) seems to remove the list notation, so that the field becomes `metadata["NWBFile"]["experimenter"] = 'name of experimenter`, which is not valid in the metadata schema. 
How should we handle this?

For the second error, we were missing an init (just pushed a [fix](https://github.com/catalystneuro/nwb-conversion-tools/pull/313/commits/b267c9a9c1f1e5336da3b63dd55f57d6b93ac4e9)).

File: nwb_conversion_tools/datainterfaces/ecephys/baselfpextractorinterface.py [hunk 1]
```diff
@@ -13,6 +13,9 @@
 class BaseLFPExtractorInterface(BaseRecordingExtractorInterface):
     """Primary class for all LFP data interfaces."""
 
+    def __init__(self, **source_data):
+        super().__init__(**source_data)
```
Comment (CodyCBakerPhD) [inline nwb_conversion_tools/datainterfaces/ecephys/baselfpextractorinterface.py:17]:
```suggestion
    def __init__(self, **source_data):
        super().__init__(**source_data)
```
What does this do that inheriting the `__init__` from the parent doesn't? Note the `AxonaLFPDataInterface` has its own override to the `__init__`, the super should just be needed there instead.

**Comment by CodyCBakerPhD** [comment]
> However, the deep_dict_update in the get_metadata of the NWBconverter class seems to remove the list notation, so that the field becomes metadata["NWBFile"]["experimenter"] = 'name of experimenter, which is not valid in the metadata schema.
How should we handle this?

@Saksham20 Is this intended behavior of the new `deep_dict_update` or another bug?

**Comment by Saksham20** [comment]
> > However, the deep_dict_update in the get_metadata of the NWBconverter class seems to remove the list notation, so that the field becomes metadata["NWBFile"]["experimenter"] = 'name of experimenter, which is not valid in the metadata schema.
> > How should we handle this?
> 
> @Saksham20 Is this intended behavior of the new `deep_dict_update` or another bug?

No, this should be fixed after merging with master which contains #315

**Comment by CodyCBakerPhD** [comment]
> No, this should be fixed after merging with master which contains #315

I've updated it with master, then. @alejoe91 Can you tell if that fixes it on your side?

File: nwb_conversion_tools/nwbconverter.py [hunk 2]
```diff
@@ -93,7 +93,7 @@ def get_metadata(self):
         metadata = get_default_nwbfile_metadata()
         for interface in self.data_interface_objects.values():
             interface_metadata = interface.get_metadata()
-            metadata = dict_deep_update(metadata, interface_metadata)
+            metadata = dict_deep_update(metadata, interface_metadata, append_list=False)
```
Comment (CodyCBakerPhD) [inline nwb_conversion_tools/nwbconverter.py:96]: If this doesn't fix the problem, can you revert it?

**Review by alejoe91 — APPROVED** [review]

**Pull request merged**